covsirphy/cleaning/jhu_data.py

```diff
@@ -558,3 +558,64 @@ def records(self, country, province=None, start_date=None, end_date=None, popula
             raise SubsetNotFoundError(
                 country=country, country_alias=country_alias, province=province,
                 start_date=start_date, end_date=end_date, message="with 'Recovered > 0'") from None
+
+    def show_complement(self, country=None, province=None,
+                          start_date=None, end_date=None, population=None,
```

Comment:
Please add test codes of 
- `JHUData.show_complement(country="Greece")`
- `JHUData.show_complement(country=["Greence", "Japan"])`
- `JHUData.show_complement(country=["Greence", "Japan"], province="Tokyo")` -> raise `ValueError`
- `JHUData.show_complement(country=None)`
in tests/test_datahub.py::TestJHUData.

Reply:
Are the following tests okay?:
```
    @pytest.mark.parametrize(
        "country, province",
        [
            ("Greece", None),
            (["Greece", "Japan"], None),
            (["Greece", "Japan"], "Tokyo"),
            (None, None),
        ]
    )
    def test_show_complement(self, jhu_data, country, province):
        if province == None:
            complement_df = jhu_data.show_complement(country, province)
            assert set(
                [JHUDataComplementHandler.AVAILABLE_COL,
                 JHUDataComplementHandler.MONOTONIC_CONFIRMED,
                 JHUDataComplementHandler.MONOTONIC_FATAL,
                 JHUDataComplementHandler.MONOTONIC_RECOVERED,
                 JHUDataComplementHandler.FULL_RECOVERED,
                 JHUDataComplementHandler.PARTIAL_RECOVERED]).issubset(complement_df.columns)
        else:
            with pytest.raises(ValueError):
                complement_df = jhu_data.show_complement(country, province)
```

Reply:
Thank you, but `if statement` could be
```Python
        if isinstance(country, list) and (province is not None):
            with pytest.raises(ValueError):
                jhu_data.show_complement(country=country, province=province)
        else:
            complement_df = jhu_data.show_complement(country=country, province=province)
            all_set = set(JHUDataComplementHandler.SHOW_COMPLEMENT_FULL_COLS)
            assert all_set.issubset(complement_df.columns)
```
(`.AVAILABLE_COL` should be included in `.SHOW_COMPLEMENT_FULL_COLS`.)